pkg/cnab/provider/parameters.go

```diff
@@ -83,3 +91,22 @@ func (d *Runtime) getUnconvertedValueFromRaw(def *definition.Schema, key, rawVal
 	}
 	return rawValue, nil
 }
+
+// getZeroValue returns the zero value for a parameter according to its type
+func getZeroValue(name string, def *definition.Schema, claim *claim.Claim) interface{} {
+	switch def.Type {
+	case "integer", "number":
```

Comment:
It doesn't really matter but I don't think `number` is possible, because of how cnab uses canonical json, right?

Reply:
Yea, I had the same thinking but wasn't sure on whether to support it here (just in case) or limit only to `integer`... could be convinced either way.  WDYT?

Reply:
How about keep it but add a comment saying it is just in case?